packages/react-composites/src/composites/common/Drawer/SpokenLanguageDrawer.styles.ts

```diff
@@ -10,12 +10,23 @@ import { _DrawerMenuStyles } from '@internal/react-components';
  */
 export const spokenLanguageDrawerStyles = (theme: Theme): _DrawerMenuStyles => ({
```

Comment:
should we always apply this fix to all drawers? (I assume all can take up the whole screen given enough options supplied)

Reply:
Applied the fix to all drawers :).